schemas/dbt_project.json

```diff
@@ -154,9 +166,36 @@
                 "+enabled": {
                     "$ref": "#/$defs/boolean_or_jinja_string"
                 },
+                "+grant_access_to": {
+                    "title": "Authorized views",
+                    "type": "array",
+                    "description": "Configuration, specific to Big Query adapter, used to setup authorized views.",
+                    "items": {
```

Comment:
If these are the only allowed options, you can also set additionalProperties: false, which helps provide hints to the autocomplete